src/App.jsx

```diff
@@ -1,25 +1,32 @@
 import { useState } from "react";
 import "./App.css";
+import City from "./components/City.jsx";
+import ZipSearchField from "./components/ZipSearchField.jsx";
 
-function City(props) {
-  return <div>This is the City component</div>;
-}
+function App() {
 
-function ZipSearchField(props) {
-  return <div>This is the ZipSearchField component</div>;
-}
+  //creating states to store the city list
+  //I want the application to rerender everytime the state of the City List changes
+  const [cityList, setCityList] = useState([])
 
-function App() {
+  
   return (
     <div className="App">
       <div className="App-header">
         <h1>Zip Code Search</h1>
       </div>
       <div className="mx-auto" style={{ maxWidth: 400 }}>
-        <ZipSearchField />
-        <div>
-          <City />
-          <City />
+        {/* ZipSearchField is modified to be able to edit the city list. It wont need to access it */}
+        <ZipSearchField setCityList={setCityList} />
+        <br />
+        <div className="results">
+          {/* this div will display the results of the zipsearch and Api call */}
+          <div id="no-results" ><h6>No Results Found</h6></div>
```

Comment:
Instead of going your route of manipulating a DOM node directly (with your `document.getElementById()...` calls), what if...

```suggestion
          {cityList.length === 0 && <h6>No Results Found</h6>}
```